javascript/event-attestator/src/ProofcastEventAttestator.js

```diff
@@ -55,7 +55,7 @@ class ProofcastEventAttestator {
     return concat([
       zeroPadValue(Buffer.from(event.account, 'utf-8'), 32),
       ...topics,
-      event.data,
+      Buffer.from(JSON.stringify(event.data), 'utf-8'),
```

Comment:
stringify needs to operate on data encoded with JSON.parse, fixed on https://github.com/pnetwork-association/pnetwork-v4/pull/71